docs/configuring-openshift.md

```diff
@@ -0,0 +1,79 @@
+- [Private container registries](#private-container-registries)
+
+  <!-- /MarkdownTOC -->
+
+<a name="updating-openshift-configuration"></a>
+## Updating OpenShift configuration
+
+  Once you have [started](#starting-openshift) OpenShift, you can view and change the master and
+  node configuration of your OpenShift cluster.
+
+  You can view the current OpenShift master configuration (_master-config.yaml_) via:
+
+  ```shell
+  $ minishift openshift config view
+  ```
+
+  For displaying the node configuration, you can specify the `target` flag.
+  For more details about the `view` command refer to its [synopsis](./minishift_openshift_config_view.md).
+
+  Let's look at [Cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) (CORS) as an example for patching the OpenShift master configuration.
+  Per default, OpenShift will only allow cross origin resource requests from the IP of the
+  cluster as well as localhost. This is specified via the `corsAllowedOrigins` property in the
+  [master configuration](https://docs.openshift.com/enterprise/3.0/admin_guide/master_node_configuration.html#master-configuration-files) (_master-config.yaml_). To change this value and allow
+  cross origin requests from all domains, one can execute:
+
+  ```
+  $  minishift openshift config set --patch '{"corsAllowedOrigins": [".*"]}'
+  ```
+
+  Per default, the master configuration is targeted, but you can also path the node config
+  by specifying the `target` flag. For more details about the
+  `set` command refer to its [synopsis](./minishift_openshift_config_set.md).
+
+  **Note:** OpenShift will be restarted after applying the patch.
+
+<a name="persistent-volumes"></a>
+## Persistent volumes
+
+Minishift supports [PersistentVolumes](https://docs.openshift.org/latest/dev_guide/persistent_volumes.html)
+of type `hostPath`. These PersistentVolumes are mapped to a directory inside the Minishift VM.
+
+The MiniShift VM boots into a tmpfs, so most directories will not be persisted across reboots (for example, when you use `minishift stop`).
+However, MiniShift is configured to persist files stored under the following host directories:
+
+* `/data`
+* `/var/lib/minishift`
+* `/var/lib/docker`
+
+Here is an example PersistentVolume config to persist data in the `/data` directory:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv0001
+spec:
+  accessModes:
+    - ReadWriteOnce
+  capacity:
+    storage: 5Gi
+  hostPath:
+    path: /data/pv0001/
+```
+
+<a name="private-container-registries"></a>
+## Private container registries
+
+To access a private container registry, follow the steps on [this page](http://kubernetes.io/docs/user-guide/images/).
+
+We recommend you to use ImagePullSecrets, but if you would like to configure access on the
+Minishift VM you can place the `.dockercfg` in the `/home/docker` directory or the `config.json`
+in the `/home/docker/.docker` directory.
```

Comment:
I never tested this. No idea whether it works. I'd be tempted to remove it, since you really should use a pull secret